src/f90/crop2mlModules.f90

```diff
@@ -0,0 +1,111 @@
+module crop2mlModules
+
+contains
+    function indice(vectorElem, elem)  ! retrieve the index of an element in a vector
+	implicit none
+        character(len=*), dimension(:):: vectorElem
+        integer::iterator, indice
+        character(len=*):: elem
+        DO iterator= 1, size(vectorElem)
```

Comment:
IF and ENDIF here are uppercase while before (other file) it was lower case.
Be consistent in your style.

Reply:
The same for DO & al.